### PR TITLE
Incorrect password hint

### PR DESCRIPTION
**Pull request by BeatC**
Restyling of Incorrect password behavior:
animation

### CODE REVIEW

File: modules/UI/invite/Invite.js [hunk 1]
```diff
@@ -62,7 +68,7 @@ class Invite {
                     // will be marked as locked.
                     if (!pass)
                         this.setLockedFromElsewhere(false);
-                    this.conference.join(this.roomLocker.password);
+                    this.conference.join(roomLocker.password);
                 });
```
Comment (damencho) [inline modules/UI/invite/Invite.js]: We can reuse pass variable here

File: modules/UI/invite/RequirePasswordDialog.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        let className = this.inputErrorClass;
+        document.getElementById(this.errorId).classList.add('hide');
+        document.getElementById(this.inputId).classList.remove(className);
+    }
+
+    /**
+     * Close the dialog
+     */
+    close() {
+        APP.UI.messageHandler.closeDialog();
```
Comment (damencho) [inline modules/UI/invite/RequirePasswordDialog.js]: We should remove this method closeDialog from message handler in general cause we are not sure which prompt we are closing (there can be other prompts that are shown).
We should just get the dialog we created with APP.UI.messageHandler.openTwoButtonDialog (assign it to variable) and use that particular dialog and call its close method.

Comment (BeatC) [inline modules/UI/invite/RequirePasswordDialog.js]: Ok, I removed it. As it was used only in one place (nameley in `RequirePasswordDialog`) I've rewritten closing logic by invoking the `close` method of dialog instance.

File: modules/UI/invite/Invite.js (hunk 1 above)
Comment (BeatC) [inline modules/UI/invite/Invite.js]: Done

**Pull request merged**